### PR TITLE
Package release commands.

### PR DESCRIPTION
**Pull request by huwshimi**
Added commands for packaging and doing releases.

### CODE REVIEW

**Comment by frankban** [comment]
:+1: ty!

File: Makefile [hunk 1]
```diff
@@ -64,6 +67,20 @@ lint: test-deps
...
 check: clean-all lint test
 
+######
+# DIST
+######
+.PHONY: sdist
+dist: clean test-deps
+	$(PYTHON) setup.py sdist
+	$(PYTHON) setup.py bdist_wheel
+	ls dist
```
Comment (bac) [inline Makefile]: The ls is a bit odd.

Comment (huwshimi) [inline Makefile]: I just copied what we had in another project, but I've removed it.

**Comment by markramm** [comment]
+1

**Pull request merged**